viroconcom/fitting.py

```diff
@@ -771,19 +769,19 @@ def _get_distribution(self, dimension, samples, **kwargs):
         # fit inspection data for current dimension
```

Comment:
Capitalize "Fit", and there are more instances in this file, which I am not all going to comment.